includes/emails/class-wc-payments-email-ipp-receipt.php

```diff
@@ -0,0 +1,256 @@
+	 * In Person Payments Receipt Email.
+	 *
+	 * An email sent to the customer when a new order is paid for with a card reader.
+	 *
+	 * @class       WC_Payments_Email_IPP_Receipt
+	 * @version     2.0.0
+	 * @package     WooCommerce\Classes\Emails
+	 * @extends     WC_Email
+	 */
+	class WC_Payments_Email_IPP_Receipt extends WC_Email {
+
+		/**
+		 * Merchant settings
+		 *
+		 * @var array
+		 */
+		public $merchant_settings = [];
+
+		/**
+		 * Charge
+		 *
+		 * @var array
+		 */
+		public $charge = [];
+
+		/**
+		 * Constructor.
+		 */
+		public function __construct() {
+			$this->id             = 'new_receipt';
+			$this->customer_email = true;
+			$this->title          = __( 'New receipt', 'woocommerce-payments' );
+			$this->description    = __( 'New receipt emails are sent to customers when a new order is paid for with a card reader.', 'woocommerce-payments' );
+			$this->template_base  = WCPAY_ABSPATH . 'templates/';
+			$this->template_html  = 'emails/customer-ipp-receipt.php';
+			$this->template_plain = 'emails/plain/customer-ipp-receipt.php';
+			$this->placeholders   = [
+				'{order_date}'   => '',
+				'{order_number}' => '',
+			];
+
+			// Content hooks.
+			add_action( 'woocommerce_payments_email_ipp_receipt_store_details', [ $this, 'store_details' ], 10, 2 );
+			add_action( 'woocommerce_payments_email_ipp_receipt_compliance_details', [ $this, 'compliance_details' ], 10, 2 );
+
+			// Triggers for this email.
+			add_action( 'woocommerce_payments_email_ipp_receipt_notification', [ $this, 'trigger' ], 10, 3 );
+
+			// Call parent constructor.
+			parent::__construct();
+		}
+
+		/**
+		 * Get email subject.
+		 *
+		 * @since  3.1.0
+		 * @return string
+		 */
+		public function get_default_subject(): string {
+			return __( 'Your {site_title} Receipt', 'woocommerce-payments' );
+		}
+
+		/**
+		 * Get email heading.
+		 *
+		 * @since  3.1.0
+		 * @return string
+		 */
+		public function get_default_heading(): string {
+			return __( 'Your receipt for order: #{order_number}', 'woocommerce-payments' );
+		}
+
+		/**
+		 * Trigger the sending of this email.
+		 *
+		 * @param WC_Order $order The order instance.
+		 * @param array    $merchant_settings The merchant settings data.
+		 * @param array    $charge The charge data.
+		 */
+		public function trigger( WC_Order $order, array $merchant_settings, array $charge ) {
+			$this->setup_locale();
+			$email_already_sent = false;
+
+			if ( is_a( $order, 'WC_Order' ) ) {
+				$this->object                         = $order;
+				$this->recipient                      = $this->object->get_billing_email();
+				$this->placeholders['{order_date}']   = wc_format_datetime( $this->object->get_date_created() );
+				$this->placeholders['{order_number}'] = $this->object->get_order_number();
+
+				$email_already_sent = $order->get_meta( '_new_receipt_email_sent' );
+			}
+
+			if ( $merchant_settings ) {
+				$this->merchant_settings = $merchant_settings;
+			}
+
+			if ( $charge ) {
```

Comment:
Just `$this->charge = $charge` please - less complexity.

Reply:
Done